libursa/Database.cpp

```diff
@@ -99,19 +94,20 @@ void Database::save() {
     db_file.open(db_base / tmp_db_name, std::ofstream::binary);
 
     json db_json;
-    db_json["config"] = {{"max_mem_size", max_memory_size}};
+    db_json["config"] = std::map<std::string, std::string>();
 
     std::vector<std::string> dataset_names;
     for (const auto *ds : working_datasets) {
         dataset_names.push_back(ds->get_name());
     }
     db_json["datasets"] = dataset_names;
 
-    json iterators_json;
+    json iterators_json = std::map<std::string, std::string>();
```

Comment:
```suggestion
    json iterators_json = std::unordered_map<std::string, std::string>();
```

This requires to `#include <unordered_map>`!